doc-content/enterprise-only/getting-started/test-suspend-proc.adoc

```diff
@@ -0,0 +1,50 @@
+[id='testing-suspend-proc']
+= Testing the driver license suspension scenario
+
+Test the driver license suspension scenario using the data that you specified when you set the Driver License Suspension rules and actions.
+
+.Prerequisites
+
+* Created the `Driver_department_traffic_violations` project
+* Created the `Violation` and `Driver` data objects
+* Set the Driver License Suspension rules and actions
+
+.Procedure
+. Log in to {CENTRAL}. Click *Menu* -> *Design* -> *Projects*, then *Driver_department_traffic_violations*.
+. Click *Add Asset* -> *Test Scenario*.
+. In the *Create new Test Scenario window* wizard, enter the following values:
+.. *Test Scenario*: `Suspend due to total points`.
+.. *Package*: select `com.myspace.driver_department_traffic_violations`.
+. Click *Ok*.
+. Click *+GIVEN* to open the *New input* window.
+. Select *Driver* from the *Insert a new fact* pull-down menu.
+. Enter `driver` in the *Fact name* field and click *Add*.
+. Click *Add a field* located under *'Driver'[driver]* to open the *Choose a field to add* window.
+. Select *totalPoints* from the *Choose a field to add* pull-down menu and click *OK*.
+. Click (image:getting-started/6191.png[]) next to *totalPoints*, click *Literal value*, then enter `10` in the *totalPoints* field.
+. Click *+GIVEN* to open the *New input* window.
+. Select *Violation* from the *Insert a new fact* pull-down menu.
+. Enter `violation` in the *Fact name* field and click *Add*.
+. Click *Add a field* located under *Insert 'Violation'[violation]* to open the *Choose a field to add* window.
+. Select *points* from the *Choose a field to add* pull-down menu and click *OK*.
+. Click (image:getting-started/6191.png[]), then click *Literal value* next to *Literal value*.
+. Enter `10` in the *points* field.
+. Click *+Expect* to open the *New expectation* window.
+. Expand the *Rule* pull-down menu, select *DriverLicenseSuspensionRule*, and click *OK*.
+. Click *+GIVEN* to open the *New input* window, enter `trafficViolation` in the *Activate rule flow group* field, and click *Add*.
+. Click *+Expect* to open the *New expectation* window and click *Add* next to *Fact value: driver*.
+. Click *Driver 'driver' has values:* to open the *Choose a field to add* window.
+. Select *state* from the *Choose a field to add* pull-down menu and click *OK*.
+. Enter `suspend` in the *state: equals* field.
+. Click *Save*, then click *Save* to confirm your changes.
+. Click *Run scenario*.
++
+
+.Result
+The rule is fired and the driver's license is suspended because the total number of points is >= 20.
```

Comment:
Why not spell out greater than instead of the symbols?

Reply:
Cause I don't want to :)